docs/docs/features/dongle.mdx

```diff
@@ -0,0 +1,206 @@
+---
+title: Keyboard Dongle
+sidebar_label: Keyboard Dongle
+---
+
+import Tabs from "@theme/Tabs";
+import TabItem from "@theme/TabItem";
+
+:::warning
+This implementation is experimental and subject to change.
+:::
+
+A bluetooth dongle can be added to any keyboard running ZMK. The result is a [split keyboard](./split-keyboards.md) with the dongle as "central". There are a number of advantages to adding a dongle, but also some disadvantages:
+
+Benefits:
+
+- The "central" dongle results in increased battery life for the keyboard/former "central", as it is now a peripheral.
+- It is easier to connect to the host device.
+
+Disadvantages:
+
+- An extra microcontroller is needed.
+- The keyboard becomes unusable without the dongle.
+
+
+
+## Editing a Keyboard Definition to Support a Dongle
+
+
+In the examples below, we will be modifying the [New Keyboard Shield](../development/new-shield.mdx) guide files and will be building on top of that.
+
+We will add the dongle in a way that it can be easily enabled, but is disabled by default.
+
+### Keyboard Shield
+
+
+Lets start by adding the dongle body to the shield configuration.
+
+```kconfig title="Kconfig.shield"
+config SHIELD_MY_KEYBOARD_DONGLE
+    def_bool $(shields_list_contains,my_keyboard_dongle)
+```
+
+### Keyboard Defconfig
+
+
+<Tabs
+defaultValue="unibody"
+values={[
+{label: 'Unibody Keyboard', value: 'unibody'},
+{label: 'Split Keyboard', value: 'split'},
+]}
+>
+<TabItem value="unibody">
+
+Next we will modify the following lines
+
+```kconfig title="Kconfig.defconfig"
+if SHIELD_MY_KEYBOARD_DONGLE
+
+    config ZMK_KEYBOARD_NAME
+        default "My Board"
+
+    config ZMK_SPLIT_ROLE_CENTRAL
+        default y
+
+    config ZMK_SPLIT
+        default y
+
+    # Prevent the dongle from going to sleep
+    config ZMK_SLEEP
+        default n
+
+    # Increase the transmit power of the dongle
+    config BT_CLTR_TX_PWR_PLUS_8
+        default y
+
+endif
+
+if SHIELD_MY_KEYBOARD
+    config ZMK_KEYBOARD_NAME
+        default "My Board"
+    
+endif
+```
```

Comment:
````suggestion
```kconfig title="Kconfig.defconfig"
if SHIELD_MY_KEYBOARD_DONGLE

    config ZMK_KEYBOARD_NAME
        default "My Board"

    config ZMK_SPLIT_ROLE_CENTRAL
        default y

    config ZMK_SPLIT
        default y

    # Prevent the dongle from going to sleep
    config ZMK_SLEEP
        default n

    # Increase the transmit power of the dongle
    config BT_CLTR_TX_PWR_PLUS_8
        default y

endif

```

Then you will need to modify the section of the `Kconfig.defconfig` where your keyboard was defined, to set it up as a peripheral when built for use with a dongle.

```kconfig title="Kconfig.defconfig"

if SHIELD_MY_KEYBOARD
    config ZMK_KEYBOARD_NAME
        default "My Board"
    
endif
```
````

You need to add the conditional to the unibody in this section. You can edit and clarify the text I suggested when doing so.

Reply:
kept it together, but split the other